openhtf/__init__.py

```diff
@@ -337,3 +379,7 @@ def __call__(self, phase_data):
       # expect it but we miscounted args, we'll get another error farther down.
       return self.func(phase_data, **kwargs)
     return self.func(**kwargs)
+
+
+# Register signal handler to stop all tests on SIGINT.
+signal.signal(signal.SIGINT, Test.HandleSigInt)
```

Comment:
It might be worth noting in the docs somewhere that we consume the SIGINT handler.